cmd/helm/repo_add.go

```diff
@@ -143,14 +145,24 @@ func (o *repoAddOptions) run(out io.Writer) error {
 	}
 
 	if o.username != "" && o.password == "" {
-		fd := int(os.Stdin.Fd())
-		fmt.Fprint(out, "Password: ")
-		password, err := term.ReadPassword(fd)
-		fmt.Fprintln(out)
-		if err != nil {
-			return err
+		if o.passwordFromStdinOpt {
+			passwordFromStdin, err := ioutil.ReadAll(os.Stdin)
```

Comment:
Is it possible to use `fmt.Scanln` here instead, or use the `io.ReadAll` instead?

Reply:
Done